pkg/flowaggregator/util/options.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 Antrea Authors
```

Comment:
I recommend renaming this file `pkg/flowaggregator/util/options/options.go`

Having a `util` package name is not considered best practice

Reply:
Fixed. Thanks.